.eslintrc.json

```diff
@@ -13,6 +14,7 @@
     "consistent-return": 0,
     "no-param-reassign": 0,
     "comma-dangle": 0,
+    "import/prefer-default-export": "off",
```

Comment:
Do not make any changes to the eslint rules. You can make the module export default an object if you have a single named export in the module